include/wget/wget.h

```diff
@@ -265,6 +265,8 @@ int
 	wget_global_get_int(int key) LIBWGET_EXPORT;
 void
 	(*wget_global_get_func(int key))(const char *, size_t) LIBWGET_EXPORT;
+void
+	wget_handle_sigwinch(void) LIBWGET_EXPORT;
```

Comment:
> Currently, the idea was to provide a library function that acts as a signal handler for SIGWINCH.

Yes, but let's assume that libwget does not know about signals (it should be OS independent). Then the app has to fetch the signal and just call wget_bar_resize(void). My idea was that this function is just a 'trigger' function. libwget should do the rest automatically, as you mention.

So basically it stays as it is, but we don't call it *sigwinch*. That name would confuse e.g. a Windows user. Maybe I just was a bit unclear.